src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -54,16 +54,21 @@ interface Props {
 }
 
 const ExerciseFinishedScreen = ({ navigation, route }: Props): ReactElement => {
-  const { exercise, results, disciplineTitle, disciplineId, documents, closeExerciseAction } = route.params
+  const {
+    exercise,
+    results,
+    disciplineTitle,
+    disciplineId,
+    documents,
+    closeExerciseAction,
+    unlockedNextExercise = false
```

Comment:
Why is this optional with default to false? The only place where nothing is set is `WriteExerciseScreen`, why? Because it is the last exercise and there is no next exercise? Why not just pass `false` there explicitly?